docs/src/fs.rst

```diff
@@ -245,10 +246,14 @@ API
 
 .. c:function:: int uv_fs_mkdtemp(uv_loop_t* loop, uv_fs_t* req, const char* tpl, uv_fs_cb cb)
 
-    Equivalent to :man:`mkdtemp(3)`.
+    Equivalent to :man:`mkdtemp(3)`. The result can be found as a null terminated string at `req->path`.
 
-    .. note::
-        The result can be found as a null terminated string at `req->path`.
+.. c:function:: int uv_fs_mkstemp(uv_loop_t* loop, uv_fs_t* req, const char* tpl, uv_fs_cb cb)
+
+    Equivalent to :man:`mkstemp(3)`. The created file path can be found as a null terminated string at `req->path`.
+    The file descriptor can be found as an integer at `req->result`.
+
+    .. versionadded:: 1.35.0
```

Comment:
This could still make it into 1.34.0.

Reply:
Sweet. I'll try to get it all ready tonight.